bioio_base/reader.py

```diff
@@ -194,6 +195,28 @@ def current_scene_index(self) -> int:
         """
         return self._current_scene_index
 
+    @property
+    def resolution_levels(self) -> Tuple[int, ...]:
```

Comment:
can't we just assume they are contiguous indices starting from zero? So all this would have to do is return a count.  I think the premise here is that they are just indexed, which is different than how scenes are handled.

Reply:
There's nothing wrong with this I suppose, but it's possibly engineering in more than what is needed.

Reply:
My thought was that if we keep it close to how scenes are used the API would be simpler/more consistent